tracim/tracim/config/app_cfg.py

```diff
@@ -274,12 +347,19 @@ def __init__(self):
             # ContentType.Folder -- Folder is skipped
         ]
 
-        self.RADICALE_SERVER_HOST = tg.config.get('radicale.server.host', '0.0.0.0')
-        self.RADICALE_SERVER_PORT = int(
-            tg.config.get('radicale.server.port', 5232)
+        self.RADICALE_SERVER_HOST = tg.config.get(
+            'radicale.server.host',
+            '0.0.0.0',
```

Comment:
listening by default on '0.0.0.0' is dangerous. Please update this with 127.0.0.1 before merge (this is a "by-the-way bugfix")